raiden/tests/utils/smoketest.py

```diff
@@ -122,32 +80,30 @@ def run_restapi_smoketests():
     assert response_json[0]['balance'] > 0
 
 
-def run_smoketests(raiden_service: RaidenService, test_config: Dict, debug: bool = False):
+def run_smoketests(
+        raiden_service: RaidenService,
+        transport: str,
```

Comment:
the genesis config file was removed, the key which configured the behavior was transport.